### PR TITLE
[New Rule] UAC Bypass via Elevated COM Interface - IEditionUpgradeManager

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/421

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Samirbous** [comment]
Example of Alert:

image

File: rules/windows/privilege_escalation_uac_bypass_com_clipup.toml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+creation_date = "2020/10/28"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/28"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies User Account Control (UAC) bypass via abusing an elevated COM Interface to launch a rogue Windows ClipUp
+program. Attackers bypass UAC to stealthily execute code with elevated permissions.
```
Comment (threat-punter) [inline rules/windows/privilege_escalation_uac_bypass_com_clipup.toml:11]:
```suggestion
Identifies attempts to bypass User Account Control (UAC) bypass by abusing an elevated COM Interface to launch a rogue
Windows ClipUp program. Attackers may attempt to bypass UAC to stealthily execute code with elevated permissions.
```

**Review by threat-punter — APPROVED** [review]
🎉

File: rules/windows/privilege_escalation_uac_bypass_com_clipup.toml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/10/28"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/28"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempts to bypass User Account Control (UAC) bypass by abusing an elevated COM Interface to launch a rogue
```
Comment (bm11100) [inline rules/windows/privilege_escalation_uac_bypass_com_clipup.toml:10]:
```suggestion
Identifies attempts to bypass User Account Control (UAC) by abusing an elevated COM Interface to launch a rogue
```

**Review by bm11100 — APPROVED** [review]
one comment, otherwise lgtm

**Pull request merged**